sdk/python/lib/pulumi/resource.py

```diff
@@ -87,6 +92,7 @@ def __init__(self,
         self.provider = provider
         self.providers = providers
         self.delete_before_replace = delete_before_replace
+        self.ignore_changes = ignore_changes
```

Comment:
Can you add a `:param` directive for this new parameter after line 87? This is what Sphinx picks up when it generates documentation.